Simplify (#64)
component-driven/component-driven-development: src/exercises/2-Design_Tokens/Readme.md

## 2.2. Accessing design tokens from components

Theme values can be accessed everywhere in the app via [ThemeProvider](https://github.com/component-driven/component-driven-development/blob/master/src/ThemeProvider.js) and the `theme` object.

> To ensure consistency of UI, always use values from the `theme` object, instead of hardcoded values.

Reviewer: ```suggestion
To ensure UI consistency, always use values from the `theme` object, instead of hardcoded values.
```

> For example following example is going to render small text with `primary` color.

Reviewer: ```suggestion
For example, render small text with the `primary` color:
```
